TestGit.Tests/Tests.cs

```diff
@@ -11,7 +11,7 @@ public void Test1()
         {
             var args = new[] {"test2", "test3"};
             var firstArg = TestGit.ParseOutFirstArg(args);
-            Assert.AreEqual(firstArg, "test2");
+            Assert.AreEqual(firstArg, "test3");
```

Comment:
I think this should be `test2`?
```suggestion
            Assert.AreEqual(firstArg, "test2");
```